lib/src/command/test_command.dart

```diff
@@ -0,0 +1,66 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+import 'package:moronepo/src/project_finder/project.dart';
+import 'package:moronepo/src/project_finder/project_finder.dart';
+
+import 'command_and_arguments_formatter.dart';
+
+class TestCommand extends Command<Null> {
+  String get workingDirectory => _fromGlobalResults("working-directory");
+
+  String get projectName => _fromGlobalResults("project");
```

Comment:
This code and above are duplicated. What do you think about create `MoronepoResults` class that will have these properties and `MoronepoCommand` that will has `moronepoResults` property?